### DISCUSSION TITLE
v19.0.3

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1038] [General]
## Releases Notes for 19.0.3
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v19.0.3/draw.io-19.0.3-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v19.0.3/draw.io-19.0.3-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v19.0.3/draw.io-universal-19.0.3.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v19.0.3/drawio-amd64-19.0.3.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v19.0.3/drawio-x86_64-19.0.3.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v19.0.3/drawio-x86_64-19.0.3.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Plugins now disabled by default. Use --enable-plugins on command line to enable.
- Updates to [draw.io core 19.0.3](https://github.com/jgraph/drawio/blob/v19.0.3/ChangeLog). All changes from 19.0.3 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v19.0.3'>v19.0.3</a>.</em>

**Comment by venkatcc** [comment]
There appears to be a bug in the integration with Mermaid API for sequence diagrams in the AppImage version 19.0.3 (I have not checked this in other installable versions). The web version does not have this issue. I'm testing with the examples from https://mermaid-js.github.io/mermaid/#/sequenceDiagram?id=critical-region and https://mermaid-js.github.io/mermaid/#/sequenceDiagram?id=break. Both of these return an error:

`Parse error on line 2: ...connection to the DB Service-->D -----------------------^ Expecting 'SOLID_OPEN_ARROW', 'DOTTED_OPEN_ARROW', 'SOLID_ARROW', 'DOTTED_ARROW', 'SOLID_CROSS', 'DOTTED_CROSS', 'SOLID_POINT', 'DOTTED_POINT', got 'NEWLINE'`

and 

`Parse error on line 4: ...ooking process fails API-->Consu -----------------------^ Expecting 'SOLID_OPEN_ARROW', 'DOTTED_OPEN_ARROW', 'SOLID_ARROW', 'DOTTED_ARROW', 'SOLID_CROSS', 'DOTTED_CROSS', 'SOLID_POINT', 'DOTTED_POINT', got 'NEWLINE'`

These errors are not seen on the web version and the sequence diagrams are rendered properly.

**Reply by davidjgraph** [reply]
can you test on app.diagrams.net?

**Reply by venkatcc** [reply]
The same examples work on app.diagrams.net.

**Reply by davidjgraph** [reply]
OK, then you just need to wait for a desktop update.

**Reply by venkatcc** [reply]
Sure thing, thanks @davidjgraph

**Comment by SuvindaPerera** [comment]
I got a little issue when i connect my drive and before clicking open
existing or create new, the window disappear no matter where you click on
the screen. I think is the natural behaviour of a popup but it's better if
we can have a confirmation step before closing or just only close the popup
when only the close button clicks. Because it is the beginning process to
create a new diagram

**Comment by SuvindaPerera** [comment]
Thank You for your response

**Comment by Witdo0192** [comment]
"Use --enable-plugins on command line to enable"
Hi, sorry if this is a stupid question, but I couldn't find the command line in draw.io desktop. How can this be done?
